tests/unit/config/param_test.c

```diff
@@ -842,6 +866,36 @@ MTF_DEFINE_UTEST_PRE(param_test, param_type_i64, test_pre)
     ASSERT_EQ(0, merr_errno(err));
 }
 
+MTF_DEFINE_UTEST_PRE(param_test, param_type_double, test_pre)
+{
+    merr_t err;
+    char   buf[128];
+    size_t needed_sz;
+
+    const struct param_spec *ps = ps_get("test18");
+
+    err = ps->ps_stringify(ps, &params.test9, buf, sizeof(buf), &needed_sz);
+    ASSERT_EQ(0, merr_errno(err));
+    ASSERT_EQ(8, needed_sz);
+    ASSERT_STREQ("0.000000", buf);
+
+    err = check(
+        "test18=12", true,
+        "test18=1.5", true,
+        "test18=-0.01", true,
+        "test18=100.", true,
+        "test18=1.1e6", false,  /* out of range */
+        "test18=-1.1e6", false, /* out of range */
```

Comment:
Did this fail because it was out of range or failed to parse? Not sure what json actually supports here.

Reply:
Intentional test for bounds:
```
        .ps_bounds = {
            .as_double = {
                .ps_min = -1e6,
                .ps_max = 1e6,
            },
        },
```

Reply:
Right, but I just want to make sure that the test actually failed due to bounds and not because it failed to parse the `+/-1.1e6`

Reply:
Good point.  I will add more test cases that answer the question.

Reply:
test cases udpated.